spring-boot-samples/spring-boot-sample-web-method-security/src/main/java/sample/security/method/SampleMethodSecurityApplication.java

```diff
@@ -64,39 +59,42 @@ public void addViewControllers(ViewControllerRegistry registry) {
 		registry.addViewController("/access").setViewName("access");
 	}
 
-	@Bean
-	public ApplicationSecurity applicationSecurity() {
-		return new ApplicationSecurity();
-	}
-
 	public static void main(String[] args) throws Exception {
 		new SpringApplicationBuilder(SampleMethodSecurityApplication.class).run(args);
 	}
 
-	@Order(Ordered.HIGHEST_PRECEDENCE)
-	@Configuration
-	protected static class AuthenticationSecurity
-			extends GlobalAuthenticationConfigurerAdapter {
-
-		@Override
-		public void init(AuthenticationManagerBuilder auth) throws Exception {
-			auth.inMemoryAuthentication().withUser("admin").password("admin")
-					.roles("ADMIN", "USER").and().withUser("user").password("user")
-					.roles("USER");
-		}
-
-	}
-
+	@EnableWebSecurity
+	@EnableGlobalMethodSecurity(securedEnabled = true)
 	@Order(SecurityProperties.ACCESS_OVERRIDE_ORDER)
```

Comment:
Thanks for pointing this out. I think we should remove `@Order` as `@EnableWebSecurity` is important as per https://github.com/spring-projects/spring-boot/pull/5052/files#r64070850